modules/pod-disruption-eviction-policy.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-configuring.adoc
+// * nodes/nodes-cluster-pods-configuring
+// * post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="pod-disruption-eviction-policy_{context}"]
+= Specifying the eviction policy for unhealthy pods
+
+When you use pod disruption budgets to specify how many pods must be available simultaneously, you can also define the criteria for how pods are considered for eviction.
+
+You can choose one of the following policies:
+
+IfHealthyBudget:: Running pods that are not yet healthy can be evicted only if the guarded application is not disrupted.
+
+AlwaysAllow:: Running pods that are not yet healthy can be evicted regardless of whether the criteria in the pod disruption budget is met.
+
+:FeatureName: Specifying the unhealthy pod eviction policy for pod disruption budgets
+include::snippets/technology-preview.adoc[]
+
+To use this Technology Preview feature, you must have enabled the `TechPreviewNoUpgrade` feature set.
+
+[WARNING]
+====
+Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone and prevents minor version updates. You should not enable this feature set on production clusters.
+====
+
+.Procedure
+
+. Create a YAML file that defines a `PodDisruptionBudget` object and specify the unhealthy pod eviction policy:
++
+.Example `pod-disruption-budget.yaml` file
+[source,yaml]
+----
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: my-pdb
+spec:
+  minAvailable: 2
+  selector:
+    matchLabels:
+      foo: bar
+  unhealthyPodEvictionPolicy: IfHealthyBudget <1>
+----
+<1> Choose either `IfHealthyBudget` or `AlwaysAllow` as the unhealthy pod eviction policy.
+
+. Create the `PodDisruptionBudget` object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f pod-disruption-budget.yaml
+----
```

Comment:
I ended up adding this as a final statement to the procedure, instead of actually adding this command as a step.

I'm assuming that draining a node is not always going to come after creating a PDB/setting this new field, so that's why I didn't want to add it. Let me know if the final statement I added looks okay.